apps/meteor/client/lib/presence.ts

```diff
@@ -51,6 +51,15 @@ const notify = (presence: UserPresence): void => {
 	}
 };
 
+declare module '@rocket.chat/rest-typings' {
+	// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+	export interface Endpoint {
+		'/v1/users.presence': {
```

Comment:
Isn't it supposed to be on `@rocket.chat/rest-typings`?

Reply:
I was considering this experimental somehow